pkg/events/configevent.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package events
+
+import (
+	"strconv"
+	"time"
+)
+
+// ConfigEvent a configuration event
+type ConfigEvent Event
+
+// ChangeID returns the changeId of the event
+func (cfgevent *ConfigEvent) ChangeID() string {
+	return cfgevent.values[ChangeID]
+}
+
+// Applied returns if the event is for an application or a rollback
+func (cfgevent *ConfigEvent) Applied() bool {
+	b, err := strconv.ParseBool(cfgevent.values[Applied])
+	if err != nil {
+		return false
+	}
+	return b
+}
+
+// CreateConfigEvent creates a new config event object
+func CreateConfigEvent(subject string, changeID string, applied bool) ConfigEvent {
```

Comment:
Thinking again the type of changeID should be change.ID
Then convert do base64.StdEncoding.EncodeToString(changeID) to make it in to string